BookingSpecTests/SampleTest.feature

```diff
@@ -3,9 +3,122 @@
 	As a math idiot
 	I want to be told the sum of two numbers
 
-Scenario: Open Main Page and Click Sign In Button
+Scenario Template: Attempt to Sign In using Invalid combinations of email and password
 	Given I open browser
 	And I navigate to url "https://booking.com"
-	Then I wait for 5 seconds
 	When I click Sign In button
```

Comment:
Let's update step 'When I click Sign In button' to 'When I click Sign In button on ribbon menu' to avoid confusions between Sign In buttons in other places.